examples/guessing.carp

```diff
@@ -14,7 +14,7 @@
       (println "(Enter q to quit.)\n")
 
       (set! guessing true)
-      (set! answer (random-between 1 100))
+      (set! answer (the Int (random-between 1 100)))
```

Comment:
This is a little obnoxious? It still works without the explicit cast, right?

Reply:
Oh, right. I don't know how that ended up there.